app/admin/companies.rb

```diff
@@ -176,5 +176,17 @@
     def scoped_collection
       super.includes(:geography, :headquarters_geography, :mq_assessments)
     end
+
+    def destroy
+      destroy_command = ::Command::Destroy::Company.new(resource.object)
+
+      results = if destroy_command.call
+                  {notice: 'Successfully deleted selected Company'}
+                else
+                  {alert: 'Could not delete selected Company'}
+                end
+
+      redirect_to admin_companies_path, results
```

Comment:
What do you think about changing `results` to `status` or `message` in all cases like this?

Reply:
👍 changed to `message` for delete, archive and batch actions